src/main/java/seedu/teachbook/ui/ClassListPanel.java

```diff
@@ -51,6 +51,7 @@ protected void updateItem(Class classObj, boolean empty) {
     }
 
     public void reload(int currentlySelectedClassIndex) {
+        classListView.getSelectionModel().clearAndSelect(-1);
```

Comment:
Not sure what is happening here, but it works... 🤔

Reply:
Oh for this why I set it to "-1" first is because if there are no changes to the selected class, the classListPanel would not update haha. So I did this "hack" to introduce a change then it somehow works.